types/post_proof_partitions.go

```diff
@@ -1,8 +1,40 @@
 package types
 
-type PoStProofPartitions uint64
+type PoStProofPartitions int
 
 const (
-	TestPoStPartitions = PoStProofPartitions(iota)
-	OnePoStPartition
+	UnknownPoStProofPartitions = PoStProofPartitions(iota)
+	OnePoStProofPartition
 )
+
+// Int returns an integer representing the number of PoSt partitions
+func (p PoStProofPartitions) Int() int {
+	switch p {
+	case OnePoStProofPartition:
```

Comment:
I assume this is because at some point >1 partition may be expected, but maybe include a comment to that extent so nobody is tempted to convert these all to if-elses.

Reply:
Yeah, we will definitely allow miners to select from a set of PoSt and PoRep proof partitions.